src/token/history/history.service.ts

```diff
@@ -1,12 +1,23 @@
-import { Injectable, NotFoundException } from '@nestjs/common'
-import { Prisma } from '@prisma/client'
-import { PrismaService } from '../../database/prisma.service'
-import { TokenHistory } from '../../token/history/entities/token-history.entity'
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { Prisma } from '@prisma/client';
+import { PrismaService } from '../../database/prisma.service';
+import { TokenHistory } from '../../token/history/entities/token-history.entity';
 
 @Injectable()
 export class TokenHistoryService {
     constructor(private prisma: PrismaService) {}
 
+    async getAllTokenHistory(where: Prisma.TokenHistoryWhereInput): Promise<TokenHistory[]> {
```

Comment:
rename ``getAllTokenHistories`` multiple

Reply:
i did it